package.json

```diff
@@ -1,11 +1,17 @@
 {
   "private": true,
   "scripts": {
-    "prepare": "lerna run prepare",
+    "prepare": "yarn -s tsref && yarn -s tsbuild && yarn -s prepare:packages && yarn -s prepare:examples",
+    "tsref": "node scripts/typescript-references.js",
+    "tsbuild": "tsc -b",
+    "tswatch": "tsc -b -w",
```

Comment:
What does this compile exactly? Shouldn't it run tswatch/tsbuild in every workspace instead? I tried to make changes in some files, adding a console.log comment in some places. I see that it is recompiling something, but pressing F5 in the browser does not pick up those changes, I still need to compile using the `yarn` command to pick up the changes (but I don't need to rebuild the browser, just pressing F5 after compile works).

Reply:
> You can also use `yarn tswatch` to watch all TypeScript files and
> rebuild when needed. It only works for TypeScript files, you still need
> to rerun `theia build` to regenerate and build the example applications.

Yes, because this new configuration only takes care of the TypeScript compilation. The bundling step still needs to be done somewhat manually. I think webpack has a watch mode too but I didn't look too much into it.

Reply:
ah ok, I can cd examples/browser and run yarn watch and then it works, thanks for the info